maxim/runnable/prompt.py

```
@@ -27,6 +33,93 @@ def __init__(self, prompt: Prompt, maxim_api: MaximAPI):
+
+                generation_result = {
+                    "id": result.id,
+                    "object": "chat.completion",
+                    "created": int(time.time()),
+                    "model": result.model,
+                    "choices": [asdict(c) for c in result.choices],
+                    "usage": asdict(result.usage),
+                }
+                generation.result(generation_result)
+
+            return result
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)